governance-charter.md

```diff
@@ -51,6 +53,50 @@ following:
 This work is to be handled by the Governance Committee or delegated to other
 project groups like the [Technical Committee](./tech-committee-charter.md).
 
+### Project Management
+
+#### Develop project proposals
+Efforts and initiatives that are too large, detailed, or important to be
+resolved in an issue or pull request are referred to as *projects*. These
+larger efforts often require collaboration between subject matter experts,
+the Technical Committee, and OpenTelemetry community members, and involve
+the creation of either a temporary or permanent SIG to accomplish their goals.
+The Governance Committee is responsible for reviewing project proposals and
+ensuring  to accomplish their goals. The [project management](https://github.com/open-telemetry/community/blob/main/project-management.md)
+document describes the process for creating and executing projects of this size.
+
+The GC is responsible for reviewing project proposals, aligning their execution
+with the wider strategy and end-user priorities, and ensuring that they have
+clear goals and are staffed sufficiently to deliver on an agreed upon deadline.
+The Governance Committee is also responsible for deciding if a project is out of
+scope for the OpenTelemetry project.
+
+For proposals that involve [donating](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#donations)
```

Comment:
Should we have a specific deliverable for this? A template maybe for a report we should fill out? Is there any set deadline for reviewing proposals?

Reply:
We should probably have a template for these reports (if only so that future GCs don't spend time re-inventing the process), though I don't think that it'll need to be linked here